tests/test_sortedset_transport.py

```diff
@@ -95,3 +95,38 @@ def test_zrem_read(self):
         with mock.patch.object(self.channel.connection, '_deliver') as mock_deliver:
             self.channel._zrem_read()
             mock_deliver.assert_called_once_with(msg, 'foo')
+
+    def test_purge(self):
+        raw_db = self.faker._db
+
+        # assert no queues exist
+        self.assertEqual(len(raw_db), 0)
+
+        # put a blank message
+        self.channel._put('foo', {})
+        # verify queue is created
+        self.assertEqual(len(raw_db), 1)
+
+        num_msg = self.channel._purge('foo')
+        # verify that we removed one message and the key in Redis does not exist
+        self.assertEqual(num_msg, 1)
```

Comment:
NIT: This isn't verifying if the key has been deleted. Either fix comment, or test.